simulariumio/tests/converters/test_cytosim_converter.py

```diff
@@ -384,11 +409,26 @@
                         "fovDegrees": DEFAULT_CAMERA_SETTINGS.FOV_DEGREES,
                     },
                     "typeMapping": {
-                        "0": {"name": "microtubule"},
-                        "1": {"name": "actin"},
+                        "0": {
+                            "name": "fiber1",
+                        },
+                        "1": {
+                            "name": "actin",
+                            "geometry": {
+                                "displayType": "FIBER",
+                                "color": "#ffc100",
+                            },
+                        },
```

Comment:
with this whole expected result, it seems like it would be good to have the same expected simularium file for all your tests, so that again, there are less changes you need to make when you update the format.

Reply:
yeah moved it to conftest, only saves one repeat